CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ As a minor extension, we also keep a semantic version for the `UNRELEASED`
 changes.
 
 
+## [0.17.0] - UNRELEASED
+
+- **BREAKING** `hydra-node` `/commit` enpoint now also accepts a _blueprint_
+  transaction together with the `UTxO` used to resolve the transaction inputs.
```

Comment:
Can we say more about what the blueprint should/must contain and what happens to it to return the draft?